app_server/app/models/Config.js

```diff
@@ -0,0 +1,13 @@
+/*
+ * Edit to your local mysql database's credentials
+ */
+
+var config = {};
+
+config.name = '';
+config.username = '';
+config.password = '';
+config.host = 'localhost';
+config.dialect = 'mysql';
```

Comment:
These may be better done via some sort of template file that's not under version control, or environment variables.